scripts/upgrade.sh

```diff
@@ -16,8 +16,8 @@
 # (c) 2015, Nolan Brubaker <nolan.brubaker@rackspace.com>
 set -eux pipefail
 
-OSAD_DIR='/opt/rpc-openstack/os-ansible-deployment'
-RPCD_DIR='/opt/rpc-openstack/rpcd'
+BASE_DIR="$(pwd)"
```

Comment:
This is brittle.  The following is more deterministic:

BASE_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd ../ && pwd )